org.eclipse.winery.edmm/src/main/java/org/eclipse/winery/edmm/plugins/PluginManager.java

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+package org.eclipse.winery.edmm.plugins;
+
+import java.io.InputStream;
+import java.lang.reflect.Constructor;
+import java.util.ArrayList;
+import java.util.List;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.parsers.DocumentBuilder;
+
+
+import io.github.edmm.core.plugin.Plugin;
+import io.github.edmm.core.plugin.PluginService;
+
+import lombok.Getter;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+import org.w3c.dom.Node;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PluginManager{
```

Comment:
Apply the appropriate Winery source code formatting. See dev guide.

Reply:
On top of that, pls make sure that you add your name to the NOTICE and .mailmap file. Further, you have to sign the ECA by Eclipse: https://www.eclipse.org/legal/ECA.php

Reply:
Can I just add myself to those files simply writing my name and email?